test/Project.toml

```diff
@@ -16,7 +16,4 @@ Cassette = "0.3.5"
 ForwardDiff = "0.10"
 MPI = "0.16, 0.17, 0.18"
 Plots = "1.16"
-StartUpDG = "0.10.1"
-StructArrays = "0.6"
-Triangulate = "2.0"
```

Comment:
You should also remove these packages from the requirements listed above

Reply:
I think we use Plots.jl in some visualization tests and need to keep the compat bounds